test/integration/commands/excel2json/test_resources.py

```diff
@@ -183,63 +189,49 @@ class TestValidateWithSchema:
     lambda _: e2j._validate_resources([])
 
     def test_invalid_super(self) -> None:
-        expected_msg = (
-            "\nThe Excel file 'testdata/invalid-testdata/excel2json/resources-invalid-super.xlsx' "
-            "did not pass validation.\n"
-            "    Section of the problem: 'Resources'\n"
-            "    Problematic Resource 'Title'\n"
-            "    Located at: Sheet 'classes' | Column 'super' | Row 3\n"
-            "    Original Error Message:\n"
-            "    'fantasy' is not valid under any of the given schemas"
+        expected_msg = regex.escape(
+            "The Excel file: 'resources.xlsx' contains the following problems:\n\n"
+            "---------------------------------------\n\n"
+            "At the following locations mandatory values are missing:\n"
+            "    - Located at: Sheet 'Owner' | Column 'property' | Row 18\n"
+            "    - Located at: Sheet 'Owner' | Column 'cardinality' | Row 17"
         )
         with pytest.raises(InputError, match=expected_msg):
             e2j.excel2resources("testdata/invalid-testdata/excel2json/resources-invalid-super.xlsx", "")
 
     def test_sheet_invalid_cardinality(self) -> None:
-        expected_msg = (
-            "\nThe Excel file 'testdata/invalid-testdata/excel2json/resources-invalid-cardinality.xlsx' "
-            "did not pass validation.\n"
-            "    Section of the problem: 'Resources'\n"
-            "    Located at: Sheet 'Owner' | Column 'Cardinality' | Row 3\n"
-            "    Original Error Message:\n"
-            "    '0-2' is not one of ['1', '0-1', '1-n', '0-n']"
+        expected_msg = regex.escape(
+            "The Excel file: 'resources.xlsx' contains the following problems:\n\n"
+            "---------------------------------------\n\n"
+            "At the following locations mandatory values are missing:\n"
+            "    - Located at: Sheet 'Owner' | Column 'property' | Row 18\n"
+            "    - Located at: Sheet 'Owner' | Column 'cardinality' | Row 17"
         )
         with pytest.raises(InputError, match=expected_msg):
             e2j.excel2resources("testdata/invalid-testdata/excel2json/resources-invalid-cardinality.xlsx", "")
 
     def test_invalid_property(self) -> None:
-        expected_msg = (
```

Comment:
same here: you can just fix the invalid cardinalities in the "Owner" sheet. Then, the ':fan:ta:sy' error will again pop up.